### PR TITLE
Final work

### PR DESCRIPTION
**Pull request by AlexandreScht**
- Recup my previous code
file prepareservice,
reset password
confirm account
cart style product
crypt data file
file mail redirecetion
-Solved bugs
Solved bug in userModel
Solved conflit

### CODE REVIEW

File: src/web/components/CartProduct.jsx
Comment (Scalpal) [inline src/web/components/CartProduct.jsx:1]: - The design on main with responsiveness is already made for this page so I really don't understand the reason of this new design 
- The design is broken / not showing properly

**Review by Scalpal — CHANGES_REQUESTED** [review]

File: src/api/db/migrations/20230228172743_initial-schema.js
[the remark this answers is not in the document]
Comment (AlexandreScht) [reply src/api/db/migrations/20230228172743_initial-schema.js:1]: this is uselull to be sure this is the account who want to reset the password and not just an person who reset password of other account

File: src/pages/mails/sent/index.jsx
[the remark this answers is not in the document]
Comment (AlexandreScht) [reply src/pages/mails/sent/index.jsx:1]: ok

File: src/web/components/CartProduct.jsx
Comment (AlexandreScht) [reply src/web/components/CartProduct.jsx:1]: the css was not in the file, i uploaded it in the next pr

**Review by AlexandreScht — COMMENTED** [review]
i fixed my pr

File: src/pages/mails/sent/index.jsx
[the remark this answers is not in the document]
Comment (Scalpal) [reply src/pages/mails/sent/index.jsx:1]: Replace text by : "We sent you an email. Please check your mailbox."

- Please read

**Review by Scalpal — CHANGES_REQUESTED** [review]

File: src/pages/mails/sent/index.jsx
[the remark this answers is not in the document]
Comment (AlexandreScht) [reply src/pages/mails/sent/index.jsx:1]: the message is changed

**Review by AlexandreScht — COMMENTED** [review]
message We sent you an email. Please check your mailbox. is done

**Review by Gautrot — APPROVED** [review]

**Review by caarstn — APPROVED** [review]

**Pull request merged**